pages/text-diff.tsx

```diff
@@ -4,16 +4,87 @@ import Box from '@mui/material/Box';
 import Button from '@mui/material/Button';
 import TextField from '@mui/material/TextField';
 import Typography from '@mui/material/Typography';
+import MenuItem from '@mui/material/MenuItem';
```

Comment:
linter errors here, run `npm run lint` to fix/check

Reply:
For some reasons, I don't see an error here when I run `npm run lint`.

Reply:
It might have just fixed it for you instead of showing an error